apps/cow-fi/app/(main)/legal/cowswap-terms/page.tsx

```diff
@@ -697,44 +696,111 @@ export default function Page() {
               modification to or deletion of a provision or part-provision under this clause shall not affect the
               validity and enforceability of the rest of these Terms.
             </p>{' '}
-            <h2 id="base-launch">COW Token Giveaway, Base Launch: Terms and Conditions</h2>
-            <h3 id="base-launch-introduction">1. Introduction</h3>
-            <p>By participating in the COW Token Giveaway, you agree to these Terms and Conditions.</p>
-            <h3 id="base-launch-eligibility">2. Eligibility</h3>
+            <h2 id="base-token-reward">
+              Terms and Conditions for Retroactive COW Token Reward for February Base Users
+            </h2>
+            <h3 id="base-token-reward-introduction">1. Introduction</h3>
             <p>
-              18+ years old. Not an employee or affiliate of the Organizer. Must have a valid compatible wallet. Must
-              follow CoW Protocol on X. Void where prohibited.
+              This is a surprise, retroactive reward program to appreciate users who traded on CoW Swap on the Base
+              network during February 2025. By being selected as a winner in this COW Token Reward, you acknowledge and
+              agree to these Terms and Conditions. This is not a typical giveaway with an entry period, but a
+              retroactive reward for past activity.
             </p>
-            <h3 id="base-launch-giveaway-period">3. Giveaway Period</h3>
-            <p>From 2024-12-12 at the time of the post on the X Platform to 2024-12-31, at 11:59 PM UTC.</p>
-            <h3 id="base-launch-how-to-enter">4. How to Enter</h3>
+            <h3 id="base-token-reward-eligibility">2. Eligibility</h3>
             <p>
-              Trade on Base using CoW Swap during the Giveaway Period. Quote Tweet the Giveaway announcement with your
-              trade's transaction hash. One entry per eligible transaction.
+              Must be a real user who conducted trades on CoW Swap on the Base Network during February 2025 and must:
             </p>
-            <h3 id="base-launch-prize">5. Prize</h3>
+            <ul>
+              <li>be minimum 18 years of age.</li>
+              <li>Not be an employee, affiliate, or immediate family member of the Organizer.</li>
+            </ul>
+            <h3 id="base-token-reward-period">3. Reward Period</h3>
```

Comment:
<ul>
              <li>be minimum 18 years of age;</li>
              <li>not be an employee, affiliate, or immediate family member of the Organizer.</li>
            </ul>